test/extension.multiline.test.ts

```diff
@@ -37,25 +37,25 @@ describe('Multiline template tests', () => {
       | \t  .lastOne(){let} >> \t\t  .lastOne()`)
 
   Test(`return template - method call (indentation - tabs)
-      | \t\tobject.call()     >> \t\treturn object.call()
-      | \t\t\t.anotherCall()  >> \t\t\t.anotherCall()
+      | \t\tobject.call()        >> \t\treturn object.call()
+      | \t\t\t.anotherCall()     >> \t\t\t.anotherCall()
       | \t\t\t.lastOne(){return} >> \t\t\t.lastOne()`)
 
   // first line gets to keep original indentation in VSCode
   Test(`return template - method call (indentation - spaces)
-      | ${indent(2)}object.call()   >> ${indent(2)}return object.call()
-      | ${indent(3)}.anotherCall()  >> \t\t\t.anotherCall()
+      | ${indent(2)}object.call()      >> ${indent(2)}return object.call()
+      | ${indent(3)}.anotherCall()     >> \t\t\t.anotherCall()
       | ${indent(3)}.lastOne(){return} >> \t\t\t.lastOne()`)
 
   Test(`return template - method call (indentation - mixed)
-      | \t\tobject.call()          >> \t\treturn object.call()
-      | ${indent(3)}.anotherCall() >> \t\t\t.anotherCall()
+      | \t\tobject.call()             >> \t\treturn object.call()
+      | ${indent(3)}.anotherCall()    >> \t\t\t.anotherCall()
       | \t\t\t.lastOne(){return}      >> \t\t\t.lastOne()`)
 
   Test(`return template - method call (indentation - completely mixed)
-      | \tobject.call()     >> \treturn object.call()
-      | \t  .anotherCall()  >> \t  .anotherCall()
-      | \t  .lastOne(){return} >> \t  .lastOne()`)
+      | \tobject.call()        >> \treturn object.call()
+      | \t  .anotherCall()     >> \t\t  .anotherCall()
+      | \t  .lastOne(){return} >> \t\t  .lastOne()`)
```

Comment:
Sorry, missed that last time but now notice you added additional tabs. Original indent was one tab only so why two here? Shouldn't that still be one tab + spaces?

Reply:
I just copied it from [let template](https://github.com/ipatalas/vscode-postfix-ts/pull/83/files/b04272e23b5d595e95b37cfbc7d396ee8b49149c#diff-c04594fbca6775313bf1259faca5b5bea36bd1cd98ef510b6458158ec292436dL34-L37) the only difference is that I replaced it with `return`